test/connectors/httpParsers/ReviewHttpParserSpec.scala

```diff
@@ -159,4 +160,57 @@ class ReviewHttpParserSpec extends WordSpec with Matchers with ReviewData {
       }
     }
   }
+
+  "The ReviewHttpParser.getAuditInfoHttpReads" when {
+    val ocelotLastUpdate: Long = 12345678
+    val auditInfo: AuditInfo = AuditInfo("pid", id, "title", 1, "author", ocelotLastUpdate, 2)
+
+    class TestSetup(status: Int, optJson: Option[JsValue] = None) {
+      private val httpMethod = "GET"
+      private val url = "/"
+      val httpResponse: AnyRef with HttpResponse = HttpResponse(status, optJson, Map())
+
+      def readResponse: RequestOutcome[AuditInfo] =
+        getAuditInfoHttpReads.read(httpMethod, url, httpResponse)
+    }
+
+    "the response is OK" when {
+      "the json successfully deserializes to the model" should {
+        "Return AuditInfo" in new TestSetup(OK, Some(Json.toJson(auditInfo))) {
+          readResponse shouldBe Right(auditInfo)
+        }
+      }
+      "the json is malformed" should {
+        "return MalformedResponseError" in new TestSetup(OK, Some(Json.obj())) {
+          readResponse shouldBe Left(MalformedResponseError)
+        }
+      }
+    }
+    "the response is BAD_REQUEST" should {
+      "return BadRequestError" in new TestSetup(BAD_REQUEST, Some(Json.obj("code" -> "BAD_REQUEST_ERROR", "message" -> ""))) {
```

Comment:
As the errors already exist you could try TestSetup(BAD_REQUEST, Json.toJson(BadRequest))

Reply:
will do